azure/Guidelines.md

```diff
@@ -747,18 +747,70 @@ The ability to retry failed requests for which a client never received a respons
 <!-- Open API Spec -->
 [OpenAPI Specification]: https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md
 
-### Long Running Operations & Jobs
+### Long-Running Operations & Jobs
 
-Azure generally follows the [Microsoft REST API guidelines for Long running operations](https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#13-long-running-operations).  Follow the Microsoft guidelines with the modifications / extensions given here when designing / implementing long running operations.
+When the processing for an operation may take a significant amount of time to complete, it should be
+implemented as a _long-running operation (LRO)_. This allows clients to continue running while the
+operation is being processed. The client obtains the outcome of the operation at some later time
+through another API call.
```

Comment:
Agreed, but we do need to give additional guidance on how to correctly model a process that takes 15+min to complete. 

In general, a long long running operation should be modeled as a discoverable (listable) resource in its own right - e.g. a "build" resource in a CI system can be "rediscovered" by listing pending builds rather than just being modeled as a generic operations resource.

Reply:
I'll take a stab at language to cover the "long-long-running operation" case.